experimental/GModule/Brueckner.jl

```diff
@@ -458,6 +474,11 @@ function lift(C::GModule, mp::Map)
   return allG
 end
 
+function solvable_quotient(G::Oscar.GAPGroup)
+  q = pc_group(symmetric_group(1))
```

Comment:
If you want a trivial pc group, perhaps this is a simpler way
```suggestion
  q = cyclic_group(1)
```

Reply:
These seem to give subtly different groups in some way (`==` returns `false`, even though they are both Pc groups of order 1....), I'll check that this change doesn't cause problems.